src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -87,6 +87,70 @@ public void afterAll() throws Exception {
     jdbcConnection.createStatement().execute(String.format("drop database %s", TEST_DB));
   }
 
+  @Test
+  public void testColumnNameQuotes() throws Exception {
+    final String tableName = "T_COLUMN_WITH_QUOTES";
+    final String quotedColumn = "\"c2\"";
+    final String doubleQuotedColumn = "\"c\"\"a\"\"3\"";
+    final String escapedSpacesColumn = "c\\ 4"; // space is escaped with '\' in Snowflake SQL
```

Comment:
Not sure if I understand this case, I tried it in the worksheet, so something like `create or replace table t_streamingingest("c\\ 1" int);` , the column name is `c\\ 1`

Reply:
One slash here is to escape it in java, the actual string is 'c\ 1'. As I understood it is to escape space for SF.

Reply:
wow, learn something new today, thanks! Do you want to add a test case with double quotes and space?

Reply:
hmm I am not sure I understand why the space is a special case for double quotes?
do you mean if user already passed the column name in quotes and used double quotes to escape the internal user quotes in the name: `doubleQuotedColumn` but with spaces inside? like:
```
"\"c\"\"a \"\" 3\"" // space right after 'a' and right before '3'
```
I can change the test case be like this and the test passes.
It does not look special, the spaces will just stay and they do not need to be escaped.
If a space is escaped in the quoted string with '\', the '\' will just stay in the string.
The double quotes is a valid escape only in an originally already quoted string iiuc.